src/lib/services/query-service.ts

```diff
@@ -79,7 +79,7 @@ export async function getQueryTypes(
   options: WorkflowParameters,
   request = fetch,
 ): Promise<string[]> {
-  return new Promise<string[]>((resolve, reject) => {
+  return new Promise<string[]>((resolve) => {
     fetchQuery(
```

Comment:
This isn't for this code change it's further up but something to consider.

```type QueryRequestParameters = {
  workflow: Eventual<{ id: string; runId: string }>;
  namespace: string;
  queryType: string;
};
type WorkflowParameters = Omit<QueryRequestParameters, 'queryType'>;```

Could be extended instead of omitted which is usually easier for me to understand